pkg/constants/constants.go

```diff
@@ -115,6 +115,12 @@ const (
 	Explain InferenceServiceVerb = "explain"
 )
 
+// Custom URLS Annotation
+const (
+	CustomURLAnnotation InferenceServiceVerb = "custom.urls"
+        CustomURLSplit      InferenceServiceVerb = ","
```

Comment:
This is not a verb, call it `CustomPredictorUrlPathSeparator`?

Reply:
accept